src/daria/corrections/shape/curvature.py

```diff
@@ -398,15 +398,41 @@ def compute_stretch(self, img: Optional[np.ndarray] = None, **kwargs):
             horizontal_stretch = 0
         # Check whether point is chosen too close to the center
         # (within 10% of total pixels), and make warning
-        elif abs(pt_src[0] - stretch_center[0]) < round(0.1 * Nx):
+        elif abs(pt_src[0] - stretch_center[0]) < round(0.05 * Nx):
             horizontal_stretch = 0
             warn(
                 "point_source chosen too close to stretch center for correction"
-                " in horizontal direction. Please choose a value towards the corners."
-                " Horizontal stretch is set to zero now."
+                " in horizontal direction (within 5"
+                "%"
+                " of pixels in horizontal "
```

Comment:
Will this be displayed in three lines?

Reply:
I will test it, but I think this is the/a way to display it without having several lines. flake8 is prohibiting me from making just one superlong string..

Reply:
Cool. Whenever you are satisfied, feel free to merge.

Reply:
It prints as expected. I just removed on whitespace to make it look nicer. Merging now.